UI/GameSettingsScreen.cpp

```diff
@@ -217,16 +217,25 @@ void GameSettingsScreen::CreateViews() {
 		return UI::EVENT_CONTINUE;
 	});
 	blockTransfer->SetDisabledPtr(&g_Config.bSoftwareRendering);
-	CheckBox *softwareGPU = graphicsSettings->Add(new CheckBox(&g_Config.bSoftwareRendering, gr->T("Software Rendering", "Software Rendering (slow)")));
-	softwareGPU->OnClick.Add([=](EventParams &e) {
-		if (g_Config.bSoftwareRendering)
-			settingInfo_->Show(gr->T("SoftGPU Tip", "Currently VERY slow"), e.v);
-		bloomHackEnable_ = !g_Config.bSoftwareRendering && (g_Config.iInternalResolution != 1);
-		return UI::EVENT_CONTINUE;
-	});
-	softwareGPU->OnClick.Handle(this, &GameSettingsScreen::OnSoftwareRendering);
-	if (PSP_IsInited())
-		softwareGPU->SetEnabled(false);
+
+	bool showSoftGPU = true;
+#if PPSSPP_PLATFORM(ANDROID)
```

Comment:
Yeah, I suppose so. MOBILE_DEVICE is a bit of an odd thing though, is the Shield TV mobile? :)  I guess it'll do the right thing in this case.